file-handling-example/src/upload_file/handler.test.ts

```diff
@@ -0,0 +1,33 @@
+import { OperationHandlerTestSetup } from "@trayio/cdk-dsl/connector/operation/OperationHandlerTest";
+import { OperationHandlerResult } from "@trayio/cdk-dsl/connector/operation/OperationHandler";
+import { uploadFileHandler } from "./handler";
+import "@trayio/cdk-runtime/connector/operation/OperationHandlerTestRunner";
+
+OperationHandlerTestSetup.configureHandlerTest(
+	uploadFileHandler,
+	(handlerTest) =>
+		handlerTest
+			.usingHandlerContext("test")
+			.nothingBeforeAll()
+			.testCase("should upload file", (testCase) =>
+				testCase
+					.givenNothing()
+					.when(() => ({
+						file_path: '/cdk-testing/meme.jpeg',
+						file: {
+							name: "meme.jpg",
+							url: "https://i.imgur.com/CKVpBov.jpg",
```

Comment:
why we have a name and a url with different file names?
should the url not only contain the url without the filename?
i don't understand this example, we have file_path and file. Not sure what this means? Maybe a comment would help?

Reply:
Go to that URL; it's an actual meme 😛

The `file` is a FileReference object - the thing passed around in workflows. 
The `file_path` is one of the arguments dropbox requires - it's where to save the file

Reply:
Also, typically the URL in the `file`/FileReference object will be a signed S3 URL, if it's coming from a workflow.

Reply:
maybe worth a comment then? (again, only because is an example repo?)